man/man8/zfs.8

```diff
@@ -4058,6 +4064,22 @@ See the documentation for
 for more details.
 .It Xo
 .Nm
+.Cm send
+.Op Fl PenvLw
+.Op Fl i Ar snapshot Ns | Ns Ar bookmark
+.Fl S
+.Ar filesystem
+.Xc
+Send a dataset that has been partially transferred via a resumable receive.
```

Comment:
What do you think of this alternate wording.

`Generate a send stream from a dataset that has been partially received.`